index.html

```diff
@@ -24,7 +24,7 @@
     <header class="navigation">
       <a class="logo" href="#">Welcome</a>
       <nav>
-        <ul class="nav_collapse" id="collapse">
+        <ul class="collapse" id="collapse">
           <li id="closebtn"><img src="./assets/images/close.png" alt="close button"></li>
           <li><a href="#recent-works">Portfolio</a></li>
           <li><a href="#about-me">About</a></li>
```

Comment:
Hi @team, first of all, good job so far working on this project 💪🏻! We however have some small changes to make before we continue to the next project.

- [ ] I have noticed that the navigation on the desktop view disappears, this makes it hard for a visitor to navigate through your portfolio website. I request that you try to fix it and make it available 😄. See image below 👇🏻.

![image](https://user-images.githubusercontent.com/4247740/147154468-c6fd1caa-0048-4e10-81bd-1fb8b6015b02.png)